docs/tutorials/install_and_admin/geonode_install/install_geonode_application.txt

```diff
@@ -1,120 +1,126 @@
-
-GeoNode Setup
-=============
-First of all we need to prepare a new Python Virtual Environment:
+    sudo -i apt update
+    sudo apt install openjdk-8-jdk-headless default-jdk-headless -y
+    sudo update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+    sudo apt install apache2 \
+        build-essential \
+        default-jdk \
+        gcc \
+        gdal-bin \
+        gettext \
+        git \
+        git-core \
+        libapache2-mod-wsgi \
+        libgdal-dev \
+        libgdal20 \
+        libgeos-dev \
+        libjpeg-dev \
+        libpng-dev \
+        libpq-dev \
+        libproj-dev \
+        libxml2 \
+        libxml2-dev \
+        libxslt-dev \
+        libxslt1-dev \
+        nano \
+        openssh-server \
+        patch \
+        postfix \
+        postgis \
+        postgresql \
+        postgresql-10-postgis-2.4-scripts \
+        postgresql-contrib \
+        python-dev \
+        python-gdal \
+        python-pastescript \
+        python-pil \
+        python-pip \
+        python-psycopg2 \
+        python-pycurl \
+        python-urlgrabber \
+        python-virtualenv \
+        software-properties-common \
+        tomcat8 \
+        unzip \
+        vim \
+        zip \
+        zlib1g-dev
+
+    sudo apt update;
+    sudo apt upgrade;
+    sudo apt autoremove;
+    sudo apt autoclean;
+    sudo apt purge;
+    sudo apt clean
+
+.. note:: The installation process is going to take several minutes and it will need to download packages.
+
+GeoNode Installation
+====================
+First of all, we need to prepare a new Python Virtual Environment:
 
 .. code-block:: bash
 
-    $ sudo apt-get install python-pip
-    $ pip install --upgrade pip
-    $ pip install --user virtualenv
-    $ pip install --user virtualenvwrapper
+    pip install virtualenv
     # The commands above will install the Python Venv packages
 
-    $ export WORKON_HOME=~/Envs
-    $ mkdir -p $WORKON_HOME
-    $ source $HOME/.local/bin/virtualenvwrapper.sh
-    $ printf '\n%s\n%s\n%s' '# virtualenv' 'export WORKON_HOME=~/Envs' 'source $HOME/.local/bin/virtualenvwrapper.sh' >> ~/.bashrc
-    $ source ~/.bashrc
-    # We have now configured the user environment
-
-    $ mkvirtualenv --no-site-packages geonode
-    # Through this command we have created a brand new geonode Virual Environment
+    mkdir ~/.venv
+    virtualenv ~/.venv/my_geonode
+    # Through this command we have created a new Virual Environment for GeoNode
 
-    $ sudo useradd -m geonode
-    $ sudo usermod -a -G geonode geo
-    $ sudo chmod -Rf 775 /home/geonode/
-    $ sudo su - geo
-    # The commands above are needed only if geo and geonode users have not been already defined
 
 Let's activate the new `geonode` Python Virtual Environment:
 
 .. code-block:: bash
 
-    $ workon geonode
-
-Move into the `geonode` home folder
-
-.. code-block:: bash
+    source ~/.venv/my_geonode/bin/activate
+    cd ~
```

Comment:
Why using `cd ~`?

Reply:
I think this can be skipped.

Reply:
@frafra This is where we start working in the users home directory. SU will most likely switch automatically to it?But I would add the cd ~ to be safe and add a line like:

> Let's activate the new `geonode` Python Virtual Environment, and make sure we're starting
> in the users home directory.